wetterdienst/dwd/observations/api.py

```diff
@@ -238,6 +294,103 @@ def collect_safe(self):
 
         return pd.concat(data)
 
+    def _invalidate_storage(self):
+        for parameter in self.parameter:
+            for period_type in self.period_type:
+                storage = self.storage.hdf5(
+                    parameter=parameter,
+                    time_resolution=self.time_resolution,
+                    period_type=period_type,
+                )
+
+                storage.invalidate()
+
+    @staticmethod
+    def _tidy_up_data(df: pd.DataFrame, parameter: Parameter) -> pd.DataFrame:
```

Comment:
Maybe we can move this into `PandasDwdExtension`?

Reply:
I wonder how this would work as we still have to hand it Parameter

Reply:
No worries. It's the same with

https://github.com/earthobservations/wetterdienst/blob/0688db2e1f3223db29692366cedbde3365c2f1bf/wetterdienst/dwd/pandas.py#L32-L34